portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +26,35 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  ArrayList<String> fun_facts = new ArrayList<String>();
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("Lifelong learner and engineer.");
+    fun_facts.add("I used to read the dictionary for fun!");
+    fun_facts.add("I considered going to the University of Madison (Better Dead Than Red)");
+    fun_facts.add("I spent over $100 on a planner in the 10th grade!");
+
```

Comment:
Since the array is Static, instead of initializing it in the doGet method, you could also make it a const and initialize it likes this
private static final List<String> FUN_FACTS = Arrays.asList("fact1", "fact2");

Since this code will change in the next step, I am ok if you want to leave it the way it is.

Reply:
I'll leave it as is, since, as you said, it will change with the next step! (Done)